src/object/srv_obj_migrate.c

```diff
@@ -2683,6 +2696,13 @@ migrate_one_epoch_object(daos_epoch_range_t *epr, struct migrate_pool_tls *tls,
 					DP_UOID(arg->oid));
 			}
 			continue;
+		} else if (rc == -DER_UPDATE_AGAIN) {
+			/* -DER_UPDATE_AGAIN means the remote target does not parse EC
+			 * aggregation yet, so let's retry.
+			 */
+			D_DEBUG(DB_REBUILD, DF_UOID "retry with %d \n", DP_UOID(arg->oid), rc);
+			rc = 0;
+			continue;
```

Comment:
my dss_sleep(10)?

Reply:
EC aggregation should be aborted soon. and also this needs resend the new RPC anyway. since rebuild is a higher priority job, so it will need retry immediately.